models/Document/Hardlink/Wrapper.php

```diff
@@ -172,9 +182,9 @@ public function hasChildren($unpublished = false)
     /**
      * @throws \Exception
```

Comment:
@return \Exception

also, could you please fix the typo in exception message? thanks!

Reply:
I fixed it